infrastructure/templates/public-api/components/functionApp.bicep

```diff
@@ -68,67 +95,63 @@ resource appServicePlan 'Microsoft.Web/serverfarms@2023-01-01' = {
   }
 }
 
-var dedicatedStorageAccountName = replace('${subscription}eessa${functionAppName}', '-', '')
+// Configuring a single shared storage account for task management, and 2 individual storage accounts to be split
+// between the production slot and staging slot. See
+// https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-zero-downtime-deployment#status-check-with-slot
```

Comment:
This is really cool. Just looking at the docs, it mentions implementing a status check function for use by the CI/CD pipeline, so that the slots are swapped only when there's no pending or running orchestrations. Don't we also need to do that here?

Reply:
I think that there's a potential decision to be made here on how to time the slot swapping.  One way is to wait until there's no activity.  The other way is to allow the Durable Functions that were originally running in the production slot to continue running while it is being swapped into the staging slot.  That way, long-running operations don't need to block deploys and, by virtue of each slot having its own dedicated storage account, it's able to continue its execution even after being swapped into staging.

I think we need to spend some time looking into these strategies, but I'm inclined not to try to add it to this particular PR.  Happy for me to spin off an additional ticket?